terraform/management/README.md

```diff
@@ -0,0 +1,19 @@
+# Management environment
+
+The "management" is a "meta" configuration, in that it specifies
+configuration pertaining to spaces and users across environments. In an
+initial bring-up/contingency recovery situation, this is the first environment
+to apply.
+
+Among other things, this module...
+- configures up the spaces for dev, staging, and production
```

Comment:
Not yet, due to provider bug; to be addressed in bootstrap PR, so ~~strikethrough~~ here until that's fixed.